src/experimental/AlgebraicNets.jl

```diff
@@ -34,7 +34,6 @@ TODO: Explain
 """
 @signature MonoidalCategoryWithBidiagonals(Ob,Hom) => AlgebraicNetTheory(Ob,Hom) begin
   linear(x::Any, A::Ob, B::Ob)::(A → B)
-  constant(x::Any, A::Ob) = (x → munit(Ob) → A)
```

Comment:
This seems really important, we need the ability to add constants into our morphisms. Should this be a definition outside the `@signiture`?

Reply:
Is constant here a type or a term? I don't know what this is supposed to be exactly. I think it needs to be rewritten as a term constructor

Reply:
Yeah I think there should be a generally better way to handle constants. the whole constant(A) = Hom(munit(Ob), A) is a real pain when declaring the axioms. Especially if we have more complex constants like the Vector thing in structured matrices. @epatters, can we do `constant(A)::Hom(munit(Ob), A)` in the theory as a term constructor and then handle them in the syntax and instances?

Reply:
The code in that lines seems to have gotten mangled during our earlier Unicode refactors. I guess I didn't have any tests to catch it :(

Anyway, yes, there should be a term constructor ``constant(A::Ob, x)::Hom(munit(), A)`` where `x` is the constant value. It can then be implemented in instances and syntax systems. We're already using this device for `scalar` terms in GLA.

Reply:
Actually, here we have ``constant(x::Any, A::Ob)::Hom(munit(), A)`` but the order isn't so important.

Reply:
Ah, that's what I thought. I have fixed it. For this entry, is there a function definition we could put in the instance `AlgebraicNetTheory`, because now it is saying there needs to be a defintion for the function `constant`

Reply:
Try something like:

```julia
function constant(x::Any, A::NFormulas)::Formulas
  @assert A.n == 1
  Formulas([x], Symbol[])
end
```

I think this only ever worked for "scalar" constants.